src/UnoFlipController.java

```diff
@@ -37,8 +33,10 @@ public UnoFlipController(UnoFlipModel model){
 
     @Override
     public void actionPerformed(ActionEvent e) {
-        if (e.getActionCommand().equals(UnoFlipViewFrame.START_CMD)){
-
+        // Start game command to initialize players
+        if (e.getActionCommand().equals(UnoFlipViewFrame.START_CMD))
+        {
```

Comment:
I think the java format for brackets is to have the open brace on the same line as the conditional, let's just stay consistent with that.